capnpy/benchmarks/test_benchmarks.py

```diff
@@ -119,6 +119,23 @@ def count_enum(obj):
         res = benchmark(count_enum, obj)
         assert res == self.N*2
 
+    @pytest.mark.benchmark(group="getattr")
+    def test_enum_imported(self, schema, benchmark):
+        if schema.__name__ != 'Capnpy':
+            py.test.skip('N/A')
+
+        benchmark.extra_info['attribute_type'] = 'enum'
```

Comment:
this should be `enum_imported`, because it's used to generate the label in the charts; e.g., in `benchmarks.rst`:

```
.. benchmark:: Get Attribute
   :foreach: b.python_implementation
   :filter: b.group == 'getattr'
   :series: b.params.schema
   :group:  b.extra_info.attribute_type
```